### PR TITLE
indents(go): improve `@branch` rules

### PR DESCRIPTION
**Pull request by theHamsta**
- Don't branch at `case`
- Let `import_spec_list`/`var_declaration` behave like
  `const_declaration`

Fixes #2166

@geefuoco can you test please!

### CODE REVIEW

File: tests/indent/go/multiple-vars.go [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+	thingTwo = 2
+)
+
+func main() {
+	// It should be
+	var (
+		thing    = 1
+		thingTwo = 2
+	)
+}
```
Comment (theHamsta) [inline tests/indent/go/multiple-vars.go:19]: this was on my PC from an old issue. essentially the same problem. I can't remember which issue it was.

**Comment by geefuoco** [comment]
Looks like its working for me. Thanks a lot !

**Review by geefuoco — APPROVED** [review]
Works as intended

**Comment by geefuoco** [comment]
I found some more indentation issues with the `select` and `select` keywords. Let me know if I should just open a new issue

**Pull request merged**